app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,85 @@
+class GroupsController < ApplicationController
+  def index
+    # currently also using this endpoint to pull down suggested groups
+    groups = if params[:limit].present? # Get a specific amount of trending
+      Group.trending(params[:limit].to_i)
```

Comment:
We should have an upper bound on this, don't want to allow people to cause us to serialize 1000 groups.